regression/tests/studio/test_studio_grading.py

```diff
@@ -50,3 +62,41 @@ def test_grade_range(self):
         self.browser.refresh()
         self.assertEquals(
             self.grading_page.letter_grade('.letter-grade'), 'Pass')
+
+    def test_assignment_types(self):
+        """
+        Verifies that user can add/delete assignment types
+        """
+        self.grading_page.click_new_assignment_type()
+        self.grading_page.fill_assignment_type_fields(
+            'Final', 'Finale', '100', '2', '1')
+        self.assertEquals(
+            self.grading_page.assignment_name_field_value(), ['Final'])
+
+        # Navigating to course outline page to see if the added assignment
+        # is available to use on subsections
+        self.studio_course_outline.visit()
+
+        self.studio_course_outline.add_section_with_name('Test Section 1')
+        self.assertIn(
+            'Test Section 1',
+            self.studio_course_outline.q(css='.incontext-editor-value').text
+        )
+
+        self.studio_course_outline.add_subsection_with_name(
+            'Test Subsection 1')
+        self.assertIn(
+            'Test Subsection 1',
+            self.studio_course_outline.q(css='.incontext-editor-value').text
+        )
+
+        self.studio_course_outline.open_subsection_settings_dialog()
+        self.assertIn(
+            'Final', self.studio_course_outline.grade_as_verification())
+
+        # Cleanup Course Outline Page
+        self.studio_course_outline.subsection_settings_cancel()
+        self.studio_course_outline.delete_section()
+        # Cleanup Grading Page
+        self.grading_page.visit()
+        self.grading_page.delete_assignment_type()
```

Comment:
@benpatterson 
We planned to add a separate ticket for addCleanup method for all tests. I think we will add it in helpers that can be used every time. 
Should I keep this for now and add it separately or do you want me to add it in this PR?